gtk2_ardour/po/fr.po

```diff
@@ -15841,7 +15867,7 @@ msgstr "L'entrée MIDI suit la sélection de piste MIDI"
 
 #: rc_option_editor.cc:3610 rc_option_editor.cc:3611
 msgid "MIDI/MIDI Port Config"
-msgstr "Configuration de Port MIDI/MIDI"
+msgstr "MIDI/Configuration de Port MIDI"
```

Comment:
It looks to me that the configuration applies to both MIDI and MIDI port
```suggestion
msgstr "Configuration MIDI & port MIDI"
```
?

Reply:
Hi and thanks for your comment.
To the best of my knowledge, "MIDI" here is not a sentence to be translated, but the location "sub-paragraph" where the real sentence to be translated is living.

Reply:
Ah, I see. Thanks for the clarification.